Makefile

```diff
@@ -5,3 +5,6 @@ generate:
 
 test: generate
 	${ts} test
+
+fmt:
+	npx prettier --tab-width 4 -w grammar.js
```

Comment:
I'm not a JS dev, so no clue if this one makes sense, I choose 4 spaces as it created only a small diff.